src/main/scala/census/db/creator/GeometryUtil.scala

```diff
@@ -0,0 +1,27 @@
+package census.db.creator
+import com.vividsolutions.jts.geom._
+import com.vividsolutions.jts.io.{WKBReader, WKTReader}
+
+object GeometryUtil {
+  val projection = 4326
+  val geometryFactory = new GeometryFactory(new PrecisionModel(), projection)
+
+  private val wktReader = new WKTReader(geometryFactory)
```

Comment:
What is wkt and wkb - can you expand the naming to help future readers of the code?